scripts/core/editor3/directive.js

```diff
@@ -160,6 +160,7 @@ class Editor3Directive {
         this.disableSpellchecker = this.disableSpellchecker || false;
         this.bindToValue = this.bindToValue || false;
         this.tabindex = this.tabindex || 0;
+        this.refreshTrigger = this.refreshTrigger || 0;
```

Comment:
Why do you need this?

Reply:
When we edit the template the value of `refreshTrigger` is undefined.